lib/modules/service.py

```diff
@@ -0,0 +1,23 @@
+import requests
+
+from data.data import Config
+
+
+class ServiceApi:
+    def __init__(self, api_url=Config.BASE_API_URL):
+        self.api_url = api_url
+
+    def service_create_entity(self, data, endpoint=Config.ENDPOINT_CREATE):
+        return requests.post(f"{self.api_url}{endpoint}", json=data)
```

Comment:
Мы не знаем какое количество аргументов будет необходимо передать в наш метод. Поэтому предлагаю отказаться от передачи в методы с запросами `data` и добавить передачу именованных аргументов `**kwargs`.
```suggestion
    def service_create_entity(self, **kwargs):
        return requests.post(f"{self.api_url}{endpoint}", **kwargs)
```